ur_robot_driver/src/hardware_interface.cpp

```diff
@@ -84,6 +84,10 @@ bool HardwareInterface::init(ros::NodeHandle& root_nh, ros::NodeHandle& robot_hw
     return false;
   }
 
+  // IP that will be used for the robot controller to communicate back to the driver.
+  std::string reverse_ip;
+  robot_hw_nh.param<std::string>("reverse_ip", reverse_ip, "");
```

Comment:
```suggestion
  std::string reverse_ip  robot_hw_nh.param<std::string>("reverse_ip", "");
```
This scheme is used everywhere around